src/gui/generated/ui/ui_SummaryWindow.ui

```diff
@@ -1,41 +1,50 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<ui version="4.0">
- <class>SummaryWindow</class>
- <widget class="QDialog" name="SummaryWindow">
-  <property name="geometry">
-   <rect>
-    <x>0</x>
-    <y>0</y>
-    <width>642</width>
-    <height>448</height>
-   </rect>
-  </property>
-  <property name="windowTitle">
-   <string>Dialog</string>
-  </property>
-  <layout class="QVBoxLayout" name="verticalLayout">
-   <item>
-    <widget class="QLabel" name="filenamelbl">
-     <property name="text">
-      <string>&quot;Filename&quot; summary</string>
-     </property>
-     <property name="alignment">
-      <set>Qt::AlignCenter</set>
-     </property>
-    </widget>
-   </item>
-   <item>
-    <widget class="QTableView" name="Dialog">
-     <property name="baseSize">
-      <size>
-       <width>3</width>
-       <height>10</height>
-      </size>
-     </property>
-    </widget>
-   </item>
-  </layout>
- </widget>
- <resources/>
- <connections/>
-</ui>
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Dialog</class>

```

Comment:
Bolo by vhodné ponechať názvy tried. Aj keď v .py súboroch si ich premenoval správne, tak pri opätovnom pregenerovaní to nebude pekné